src/data/ignore.json

```diff
@@ -27,12 +30,18 @@
     "EPUB 3 Working Group": {
       "comment": "specs not targeted at browsers"
     },
+    "Internationalization Working Group": {
```

Comment:
I don't think that's a broadly true stable statement; I think I would prefer the specs to be labeled as non-browser individually.

Reply:
Which of the specs that the group develop would you qualify as browser specs?
https://www.w3.org/groups/wg/i18n-core/publications

It seems to me that a browser spec is going to be the exception more than the rule.

But, as above, this is currently only meant to work today and likely going to evolve. I'm not even sure that we'll have that many groups that we can flag as "non browser". I'm fine dropping that for now and to rather hardcode the exception in `specs.json`.